native/helpers/challenges.js

```diff
@@ -100,6 +100,16 @@ export const challenges = [
     tags: [],
     isNew: true,
   },
+  
+  {
+    title: 'Bricker Breaker Game',
+    link: 'brick_breaker',
+    difficulty: 'medium',
+    developer: '5h0ov',
+    tags: [],
+    youtube: 'https://www.youtube.com/channel/UCYWgCjYT8_05-rLq0Xmbm-Q',
```

Comment:
only video link of the solution to the challenge is allowed. Please keep it empty if there is no link

Reply:
oh i see oops sorry hold on

Reply:
done sir @sadanandpai

Reply:
@5h0ov The experience is completely broken for mobile users